src/runtime/crt/host/main.cc

```diff
@@ -110,7 +110,7 @@ tvm_crt_error_t TVMPlatformGenerateRandom(uint8_t* buffer, size_t num_bytes) {
 }
 }
 
-uint8_t memory[512 * 1024];
+uint8_t memory[2048 * 1024];
```

Comment:
needed?

Reply:
To run tinlymlperf models right out of the box, we need to increase it.

Reply:
ok, may be worth submitting that as a separate change

Reply:
sure, changed it.